### PR TITLE
Solves ghosts re-entering shutdown drones by making the ghost unable to reenter corpses

### PR DESCRIPTION
**Pull request by warior4356**
**What does this PR do:**
When a drone dies the player is ghosted and is tagged with the inability to enter the corpse. There is a small "quirk" in this fix. You are unable to join as a golem or posibrain. I am open to suggestions on better ways to fix this. 

**Changelog:**
:cl: Warior4356
fix: Ghosts can no longer re-enter a drone that has been shut down.
/:cl:

### CODE REVIEW

**Comment by tigercat2000** [comment]
The proper solution here is not gibbing the drone, it's making it so they can't reenter a dead one.

**Comment by warior4356** [comment]
That was my first thought @tigercat2000 But looking at the description in the drone, it really sounds like the intent was for the drone to be in pieces. What is the benefit of a off drone just stitting there dumbly compared to a very clear pile of robot gibs.

**Comment by KasparoVy** [comment]
De-ja vu. I could swear this very thing was PR'd and shot down before

**Comment by Spartan6** [comment]
> De-ja vu. I could swear this very thing was PR'd and shot down before

Or perhaps it was gibbing on emag. Probably the latter


The latter.

**Comment by warior4356** [comment]
An additional argument for this solution to the problem. Drones are supposed to avoid crew. If the random crew can get the FUN! of swiping a drone and making it explode then they will be more motivated to blow up drones interacting with the crew. This will then incentivize drones stay away from crew more.

TLDR: More FUN for crew, more RP for drones.

**Comment by KasparoVy** [comment]
The whole not being able to join as x,y,z should've definitely been in the changelog.
